src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1710,6 +1710,7 @@ fn new_graph_editor(app:&Application) -> GraphEditor {
 
     frp::extend! { network
         eval_ touch.background.selected(model.breadcrumbs.outside_press.emit(()));
+        eval_ inputs.edit_mode_on(model.breadcrumbs.outside_press.emit(()));
```

Comment:
Could you do it by merging flows earlier, please? Like:
```rust
deactivate_breadcrumbs <- any_(&touch.background.selected,&inputs.edit_mode_on);
eval_ deactivate_breadcrumbs(model.breadcrumbs.outside_press()));
```

Reply:
Done